### PR TITLE
Initial commit of 2022-05-29

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

**Review by miwithro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of May.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+
+### Release notes
+
+* Features
+  * AKV with Private Link with KMS is now supported
```
Comment (shashankbarsin) [inline CHANGELOG.md:18]: let's expand AKV to Azure Key Vault. Also is there any doc link for this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-05-29
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-06-10 for public cloud and 2022-06-13 for sovereign clouds.
+
+### Announcements
+* From Kubernetes 1.23, containerd will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation [here](https://docs.microsoft.com/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview).
```
Comment (palma21) [inline CHANGELOG.md:8]: Time to remove this or not yet?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+### Release notes
+
+* Features
+  * AKV with Private Link with KMS is now supported
+  * Preview of Kubernetes 1.24
+* Bug fixes
+  * Add extra information in error messages when a subnet is full or drain issues are found
+* Component Updates
+  * Upgrade Azure File CSI driver to v1.18.0
```
Comment (shashankbarsin) [inline CHANGELOG.md:23]: can we add this link - https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.18.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+* Starting with Kubernetes 1.24, the following changes will be made:
+  * The default format of clusterUser credential for AAD enabled clusters will be ‘exec’, which requires [kubelogin](https://github.com/Azure/kubelogin) binary in the execution PATH. If you are using Azure CLI, it will prompt users to download kubelogin. There will be no behavior change for non-AAD clusters, or AAD clusters whose version is older than 1.24. Existing downloaded kubeconfig will still work. We provide an optional query parameter ‘format’ when getting clusterUser credential to overwrite the default behavior change, you can explicitly specify format to ‘azure’ to get old format kubeconfig.
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of May.
```
Comment (palma21) [inline CHANGELOG.md:12]: We're in June what's the update on this?
cc @miwithro @alvinli222

**Review by palma21 — APPROVED** [review]

**Pull request merged**